### ISSUE TITLE
Suggestion to improve map resolution and possibly reducing load times

### ISSUE THREAD

**Issue by defiancecp** [defiancecp/GTBikeVRouteDB#29]
Suggestion per Cassio Davi: "You can use tiles and various zoom levels instead of one large image, there's a sample code here https://github.com/.../gta-5/osm-gpx-embed/osm-gpx-embed.php and you can see it working here http://so2.club:8282/fotos/index.php/2020/01/27/teste-4/ it's simple and you can zoom in/out to see where it's located"

which, if I use the 8k files as a baseline and load only relevant portions, seems like it would improve load times for most maps (may make them worse for very far zooms though?) and allow for better detail levels.

**Comment by defiancecp** [comment]
lost some of the link data in that first comment, needed info is here:
https://gdal.org/programs/gdal2tiles.html?fbclid=IwAR2_o1uq5d4TrAVpoffjhvob4jhJIGLFm1IkBv4yMIARcJpSdGBuSMHropU

**Comment by oldnapalm** [comment]
> (may make them worse for very far zooms though?)

Not if you use multiple zoom levels. The sample code uses levels 3 to 7
https://github.com/oldnapalm/osm-gpx-embed/blob/gta-5/osm-gpx-embed/osm-gpx-embed.php

Zoom 3 = 3 x 3 tiles of 256 x 256 px
Zoom 7 = 43 x 43 tiles of 256 x 256 px

**Comment by defiancecp** [comment]
That makes sense.  This is the next enhancement I plan to work on, just haven't had time this past few weeks.  Every time I look at one of the really small rides (like a few of the BMX ones), that super-blurry text makes me grumpy :P

**Comment by oldnapalm** [comment]
Yes, it's ugly for the short routes. Other than that the route preview is great with the animation and mouseover for the ride data. Excellent work, congratulations.
